tests/tasks/test_dao_complete_task.py

```diff
@@ -37,6 +38,19 @@ def test_achieve_not_existent_task(self):
 
         self.assertEqual(expected_response, actual_response)
 
+    def test_achieve_task_not_in_relation(self):
+        task_id = 1
+
+        self.assertFalse(self.tasks_list_2.find_task_by_id(task_id).get("is_done"))
+        expected_response = (
+            messages.MENTORSHIP_RELATION_DOES_NOT_EXIST,
+            HTTPStatus.NOT_FOUND,
+        )
+
+        actual_response = TaskDAO.complete_task(self.first_user.id, 123123, task_id)
```

Comment:
also could we be a little more explicit about what arguments we are calling. I had to check the code to understand what the middle argument was. I know most of the code does not do this, but its something we can improve for sure.
```
TaskDAO.complete_task(
    user_id=self.first_user.id,
    mentorship_relation_id=123123,
    task_id=task_id
)
```

@devkapilbansal let me know what you think